bundle.md

```diff
@@ -1,6 +1,6 @@
-# Filesystem Bundle
+# <a name="filesystemBundle" />Filesystem Bundle
 
-## Container Format
+## <a name="containerFormat" />Container Format
```

Comment:
The proposed PR separates the anchor tag from the header text.  

This follows the recommendation from the Open API Initiative (formerly Swagger) which.

I'm hesitant to use the auto-generated anchor links from GitHub since we might change the header text (ex: adding section numbers like "4.1 Container Format" and that would change the auto-generated anchor links.